tests/framework/objects/environment_manifest.go

```diff
@@ -31,15 +31,15 @@ type EnvironmentManifest struct {
 	LoadTime           int
 }
 
-//NewManifest creates a new instance of EnvironmentManifest
-func NewManifest() EnvironmentManifest {
-	e := EnvironmentManifest{}
-	flag.StringVar(&e.HostType, "host_type", "localhost", "Host were tests are run eg - localhost,GCE or AWS")
-	flag.StringVar(&e.RookImageName, "rook_image", "rook/rook", "Docker image name for the rook container to install, must be in docker hub or local environment")
-	flag.StringVar(&e.ToolboxImageName, "toolbox_image", "rook/toolbox", "Docker image name of the toolbox container to install, must be in docker hub or local environment")
-	flag.StringVar(&e.SkipInstallRook, "skip_install_rook", "false", "Indicate if Rook need to installed - false if tests are being running at Rook that is pre-installed")
-	flag.IntVar(&e.LoadConcurrentRuns, "load_parallel_runs", 20, "number of routines for load test")
-	flag.IntVar(&e.LoadVolumeNumber, "load_volumes", 1, "number of volumes(file,object or block) to be created for load test")
-	flag.IntVar(&e.LoadTime, "load_time", 1800, "number of seconds each thread perform operations for")
-	return e
+var E EnvironmentManifest
```

Comment:
Could you call it `Env`? Even better if we could add this to the test context instead of creating a global var